examples/simple_example/variables.tf

```
@@ -14,12 +14,26 @@
  * limitations under the License.
  */
 
-variable "project_id" {
-  description = "The ID of the project in which to provision resources."
-  type        = "string"
+variable "parent_id" {
+  description = "Id of the resource under which the folder will be placed."
 }
 
-variable "bucket_name" {
-  description = "The name of the bucket to create."
-  type        = "string"
+variable "parent_type" {
+  description = "Type of the parent reosurce, defaults to organization."
+  default     = "organization"
 }
+
+variable "names" {
+  description = "Folder names."
+  default     = []
+}
+
+variable "per_folder_admins" {
+  description = "List of comma delimited IAM-style members per folder who will get extended permissions."
```

Lists are always comma delimited.

```suggestion
  description = "List of IAM-style members per folder who will get extended permissions."
```